test/evergreen/code_change_report/code_change_info.py

```diff
@@ -140,22 +295,59 @@ def create_report_info(change_list: dict, coverage_data: dict):
                 line_info['old_lineno'] = line.old_lineno
 
                 if line.new_lineno > 0:
-                    line_info['count'] = find_line_coverage(coverage_data=coverage_data, file_path=new_file, line_number=line.new_lineno)
-                    line_info['branches'] = find_covered_branches(coverage_data=coverage_data, file_path=new_file, line_number=line.new_lineno)
+                    line_coverage = find_line_coverage(coverage_data=coverage_data,
+                                                       file_path=new_file,
+                                                       line_number=line.new_lineno)
+                    branch_coverage = find_covered_branches(coverage_data=coverage_data,
+                                                            file_path=new_file,
+                                                            line_number=line.new_lineno)
+                    line_info['count'] = line_coverage
+                    line_info['branches'] = branch_coverage
+                    if line.old_lineno < 0:
+                        # Ihe line was changed, so update the counts for the overall change
+                        if line_coverage >= 0 and is_useful_line(line.content):
+                            change_num_lines += 1
+                            if line_coverage > 0:
+                                change_num_lines_covered += 1
+                        change_num_branches += len(branch_coverage)
+                        change_num_branched_covered += get_num_branches_covered(branch_coverage_info=branch_coverage)
```

Comment:
Well spotted on the typo!

Rather than removing the get_num_branches_covered function, I've replaced it's implementation with your (simpler) expression.